doc/usecases/map_honors.md

```diff
@@ -0,0 +1,24 @@
+# Use Case: Giving Map Honors
+
+## Summary
+
+- **Scope:** Map Honors
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The users can give an honor/like to the map of another user they visit.
+- **Status:** Draft
+
+## Scenarios
+
+- **Precondition:**
+  - The user has opened the app.
+  - The user is currently viewing the map of another user.
+- **Main success scenario:**
+  - The user uses the honor/like button to honor/like the currently visited map.
```

Comment:
If they are already inside the map, they have opportunities to leave likes in the Photos layer. How does this use case differ?

Reply:
To be honest, I seem to have missed the use case that already proposes a 'like' feature. In which branch can I find that feature if I may ask?
Regardless, if the feature already exists then my use case is justifiably obsolete, because there would be probably not a lot of difference. Though I will double check with the other use case to be completely sure of that or if there may be potential for two different use cases.

Reply:
Not yet merged, it is in #74 [doc/usecases/layers/photo_layer.md](https://github.com/ElektraInitiative/PermaplanT/pull/74/files#diff-73784c36b31f25b17fca52d012eb61fb252301b4631b301b4a0c9e0c6824cf3a), please review this PR with special attention how the use cases could be improved with respect to gamification.

> Regardless, if the feature already exists then my use case is justifiably obsolete

Imho it makes sense to have a "like/star/blossoms/..." on both individual places of the map and the whole map. So I think this use case make sense if you rephrase it on the whole map. The use cases might even interact, as likes on individual spots of the map probably should lead to likes of the whole map.

See doc/usecases/map_search.md for a related use case (probably the ones who are liked more, should be higher ranked).

Reply:
I'm not sure if it is a bug on side of GitHub or if it has something to do with missing permissions on my part, but I can't seem to find any trace of a like feature in the linked PR and use case. For me it only states, that a user can add/delete/move a picture and open a chat on an uploaded photo.

Reply:
I agree, it is not so obvious with how the use case is written at the moment, I made suggestions. Please comment over there how you would improve so that it gets plays more into gamification.

Reply:
Okay, now I see what you meant with the feature on the photo layer. Thanks for your clarification.

> ..., as likes on individual spots of the map probably should lead to likes of the whole map.

I don't really agree with the specific wording, because a person could like one specific aspect of the map but that does not necessarily mean he/she like the entire map. But I can imagine, that an average of likes on specific pictures in relation to the overall number of pictures on the map could be used to influence the like count on the whole map.

Reply:
Yes, I agree: there is a difference in liking specific spots and the whole map. But also if only specific spots are liked, the map can be ranked higher in the search.